lib/delayed/engine.rb

```diff
@@ -1,5 +1,7 @@
 module Delayed
   class Engine < Rails::Engine
+    engine_name 'delayed'
```

Comment:
Necessary for the `delayed:install:*` rake tasks to work. Else, it's `rake delayed_engine:*` which is meh.